src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-ProfileCard/styles.ts

```diff
@@ -0,0 +1,180 @@
+import styled from '@emotion/styled';
+import * as C from '../../ignitus-Atoms/colors';
+import { flexibleColDiv, flexibleRowDiv, TextIconContainer } from '../../shared';
+import ChatMessageIcon from '../../ignitus-Assets/ignitus-Icons/chatMessageIcon/chatMessageIcon';
+import FileIcon from '../../ignitus-Assets/ignitus-Icons/fileIcon/fileIcon';
```

Comment:
Don't import every Icon, use this approach defined here : https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-AboutPage/ignitus-CoreTeam/Styles/index.ts#L100